king_phisher/client/application.py

```diff
@@ -552,6 +554,13 @@ def do_config_load(self, load_defaults):
 			for key, value in client_template.items():
 				if not key in self.config:
 					self.config[key] = value
+			env_proxy = os.environ['HTTPS_PROXY']
+			if env_proxy is not None:
+				if env_proxy != self.config['proxy.url']:
+					self.logger.warning('Proxy environment variable does not match the user configuration.'
```

Comment:
The indentation is off here. We actually have a note in the [style guide](https://king-phisher.readthedocs.io/en/latest/development/style_guide.html#multi-line-indentation) about how it should be done for consistency. Also just about every other log line is not a full and complete sentence but instead starts with a lower case letter (except for acronyms) and does not end with a period.